codec-http/src/main/java/io/netty/handler/codec/http/CookieUtil.java

```diff
@@ -16,9 +16,15 @@
 package io.netty.handler.codec.http;
 
 
+import java.util.BitSet;
+
 import io.netty.util.internal.InternalThreadLocalMap;
+import io.netty.util.internal.logging.InternalLogger;
+import io.netty.util.internal.logging.InternalLoggerFactory;
+
+final class CookieUtil {
 
-final class CookieEncoderUtil {
+    private static final InternalLogger LOGGER = InternalLoggerFactory.getInstance(CookieUtil.class);
```

Comment:
I don't get you?! It's `static final` so it's a constant, even if it's a private one.